pkg/json/lib/json.dart

```diff
@@ -624,6 +634,8 @@ mixin _ToJson on _Shared {
           ]);
         case 'int' || 'double' || 'num' || 'String' || 'bool':
           return valueReference;
+        case 'DateTime':
+          return RawCode.fromParts([valueReference, '.toIso8601String()']);
```

Comment:
It would be more readable if `Code.toString()` allowed `RawCode.fromString('$valueReference.toIso8601String()')`